app/experimenter/experiments/models.py

```diff
@@ -267,8 +268,39 @@ def bugzilla_url(self):
 
     @property
     def monitoring_dashboard_url(self):
-        if self.is_begun and self.normandy_slug:
-            return settings.MONITORING_URL.format(slug=self.normandy_slug)
+        start_date = ""
+        end_date = ""
+
+        if (
+            self.status == self.STATUS_LIVE
+            or self.status == self.STATUS_COMPLETE
+        ) and self.normandy_slug:
+            start_date = (
+                int(
```

Comment:
Ack!  `black` had a field day with this XD

Try splitting this up a bit so it doesn't get spread out so much. 

You can define a local function inside this function for the conversion:

    def to_timestamp(date):
        return int(time.mktime(date.timetuple())) * 1000

and then just call that here and that should make things a little cleaner.